league/models.py

```diff
@@ -1108,6 +1109,7 @@ def check_ogs(self, opponents):
                 else:
                     continue
                 if game_ended < time_limit:
+                    # why not use continue statement ?
```

Comment:
Because if `game_ended < time_limit` we want to stop looking.
https://github.com/climu/openstudyroom/blob/c38842dd5916e59102254d6a04ba4a8a9271860c/league/models.py#L1157-L1162